manifest.yml

```diff
@@ -13,3 +13,23 @@ applications:
     - registers-frontend
     # environment variables (persisted for blue/green deploys)
     - registers-product-site-environment-variables
+- name: registers-frontend-queue
+  memory: 1GB
+  instances: 1
+  command: bundle exec 'rake environment jobs:work'
```

Comment:
Does this work okay having another `command` on line 29?

Reply:
They are separate artefacts. Every `- name:` is a different app, then there is one commant for each artefact.

Reply:
This PR changes the manifest to deploy multiple separate apps in CF as per: https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#multi-apps
They are all sharing the same artefact and services but with different `command` entry points. You can see this setup deployed in `sandbox`.